controllers/users.controller.js

```diff
@@ -0,0 +1,52 @@
+const { password } = require("pg/lib/defaults");
+const db = require("../models");
+const jwt = require("jsonwebtoken");
+const users = db.users;
+
+exports.createUser = (req, res) => {
+  if ((!req.body || !req.body.name || !req.body.email || !req.body.password)) {
+    res.status(400).send({
+      message: "User cannot be empty"
+    });
+    return;
+  }
+  const user = {
+    name: req.body.name,
+    email: req.body.email,
+    password: req.body.password,
+  }
+  users.create(user)
+    .then((user) => {
+      const accessToken = jwt.sign({ user: user}, process.env.TOKEN_SECRET_KEY, { expiresIn: '1h' });
+      res.json({accessToken});
+    })
+    .catch((err) => {
+      res.status(500).send({
+        message: err.message || "Some error occurred"
+      });
+    });
+};
+
+exports.findUser = (req, res) => {
```

Comment:
What's the purpose of this method?

Reply:
Its checks a whether a user exists, for login

Reply:
If its a helper method, then it shouldn't be in controller